File: Basic Programming/Hackerrank 30 days of code python3 solution/Day_2.py
#                      Name : Arif Khan
#                      Judge: HACKERRANK
#                      University: Primeasia University
#                      problem: Day 2: Operators
#                      Difficulty: Easy
#                      Problem Link: https://www.hackerrank.com/challenges/30-operators/problem
#

def get_total_cost_of_meal():
    # original meal price
    meal_cost = float(input())
    # tip percentage
    tip_percent = int(input())
    # tax percentage
    tax_percent = int(input())

    # Write your calculation code here
    tip = (meal_cost * tip_percent)/100
    tax = (meal_cost * tax_percent)/100

    # cast the result of the rounding operation to an int and save it as total_cost 
    total_cost = int(round(meal_cost+tip+tax))
    
    return str(total_cost)

# Print your result
print("The total meal cost is " + get_total_cost_of_meal() + " dollars.")

File: Basic Programming/Hackerrank 30 days of code c++ solution/Day_2.cpp
/*                    Name : Arif Khan
                      Judge: HACKERRANK
                      University: Primeasia University
                      problem: Day 2: Operators
                      Difficulty: Easy
                      Problem Link: https://www.hackerrank.com/challenges/30-operators/problem
*/

#include <bits/stdc++.h>
using namespace std;

int main() {
    double p , tax ,tip,result;
    int meal,t;
    cin >> p >> meal >> t;
    tip = (p *meal)/100;
    tax = (p *t)/100 ;
    result = p + tip + tax ;
 
     if(result<(int)result+.45)
     cout << "The total meal cost is "<<(int)result <<" dollars.\n";
     else
     cout << "The total meal cost is "<<(int)result+1 <<" dollars.\n";
    return 0;
}